app/templates/components/forms/admin/content/social-links-form.hbs

```diff
@@ -21,10 +21,10 @@
     }}
   </div>
   <div class="field">
-    <label>{{t 'Google +'}}</label>
+    <label>{{t 'Google Groups'}}</label>
```

Comment:
ok sir .

Reply:
Why Plural?
`Groups` implies you can add multiple groups, but there is only one text field. Keep it `Group`